### ISSUE TITLE
Error: failed to create containerd task

### ISSUE THREAD

**Issue by djjudas21** [galexrt/container-node_exporter-textfiles#17]
I'm running into this error while deploying `node_exporter-textfiles`:

```
Error: failed to create containerd task: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:75: mounting "/var/snap/microk8s/common/var/lib/kubelet/pods/66d75cbb-0865-4bd3-a995-5a53ceba006b/containers/smartmon/ff83452c" to rootfs at "/dev/termination-log" caused: open /var/snap/microk8s/common/run/containerd/io.containerd.runtime.v2.task/k8s.io/smartmon/rootfs/dev/termination-log: read-only file system: unknown
```

It's a 3-node physical cluster running MicroK8s. The node OS is Ubuntu LTS and `smarttools` is installed.

My `values.yaml`:

```yaml
textfiles:
  # List of textfiles script that should be run in the daemonset as separate containers
  - name: smartmon
    script: smartmon.py
    # If unset, will default to the `name`
    outputFilename: "smartmon"
    # Defaults to `300`
    interval: 900
    additionalEnv: []
    additionalVolumeMounts:
      - mountPath: /dev
        name: host-dev
        readOnly: true
    resources: {}
```

**Comment by djjudas21** [comment]
Ah, it might be because I have mounted `/dev` from the host so it can read SMART metrics, but MicroK8s wants the pod to log to `/dev/termination-log` which is obviously mounted ro

**Comment by djjudas21** [comment]
As a test, I manually set `spec.template.spec.containers.terminationMessagePath` to anything outside of `/dev` (in this case, I set `/termination-log`) and the pod started up successfully.

I think the solution to this problem is to expose `spec.template.spec.containers.terminationMessagePath` in the Helm chart so it can be overridden

**Comment by galexrt** [comment]
@djjudas21 Fixed in Helm chart version 0.1.3. The "default" for textfile containers is now `/tmp/termination-message` so that it should not conflict with `/dev` mount.
